pkg/storageincentives/agent.go

```diff
@@ -35,11 +35,19 @@ const loggerName = "storageincentives"
 const (
 	DefaultBlocksPerRound = 152
 	DefaultBlocksPerPhase = DefaultBlocksPerRound / 4
+
+	// min transactions required for each game
+	minTxCountToCover = 25
```

Comment:
the comment should be // min # of transactions our wallet should be able to cover
also on second thought 25 is too much, it should be 5.
if a node wins a round, then there are a total of 3 trxs. We leave a buffer of 2 trxs.